release/xgboost_tests/run.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
```

Comment:
This is a basic question, but where is this script actually run? Is it handled by the releaser tool?

Reply:
Yes, the releaser tool calls this!